addons/pos_self_order/static/src/app/pages/cart_page/cart_page.js

```diff
@@ -4,19 +4,19 @@ import { useSelfOrder } from "@pos_self_order/app/services/self_order_service";
 import { PopupTable } from "@pos_self_order/app/components/popup_table/popup_table";
 import { _t } from "@web/core/l10n/translation";
 import { OrderWidget } from "@pos_self_order/app/components/order_widget/order_widget";
-import { SlotsPopup } from "@pos_self_order/app/components/slots_popup/slots_popup";
+import { PresetInfoPopup } from "@pos_self_order/app/components/preset_info_popup/preset_info_popup";
 
 export class CartPage extends Component {
     static template = "pos_self_order.CartPage";
-    static components = { PopupTable, OrderWidget, SlotsPopup };
+    static components = { PopupTable, OrderWidget, PresetInfoPopup };
     static props = {};
 
     setup() {
         this.selfOrder = useSelfOrder();
         this.router = useService("router");
         this.state = useState({
             selectTable: false,
-            selectSlots: false,
+            fillInformations: false,
```

Comment:
Does `fillInformations` mean "Are the information filled/complete?"? If so, then rename it to `informationComplete` or `infoFilled`.

Reply:
That's the state to know if we need to display the info popup or not

Reply:
I see.